aiida/manage/configuration/settings.py

```diff
@@ -10,7 +10,7 @@
 """Base settings required for the configuration of an AiiDA instance."""
 import os
 import pathlib
-import typing
+from typing import Optional
```

Comment:
let's use `from __future__ import annotations` instead and the `| None` syntax

Reply:
Cool, I wasn't sure what's the policy on this. Done.